test/hybrid/server.dart

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:async/async.dart';
+import 'package:http/src/utils.dart';
+import "package:stream_channel/stream_channel.dart";
+
+/// The list of headers to ignore when sending back confirmation.
+final _ignoreHeaders = <String>[
+  // Browser headers (Chrome)
+  'accept',
+  'accept-language',
+  'accept-encoding',
+  'connection',
+  'origin',
+  'referer',
+
+  // Dart IO headers
+  'cookie',
+  'host',
+];
+
+hybridMain(StreamChannel channel) async {
```

Comment:
Document this. In particular, mention which tests it's used in and what the server does.